openassessment/workflow/models.py

```diff
@@ -731,9 +731,6 @@ def start_workflow(cls, team_submission_uuid):  # pylint: disable=arguments-diff
         team_assessment_api = team_staff_step.api()
         team_assessment_api.on_init(team_submission_uuid)
 
-        team_workflow.status = TeamAssessmentWorkflow.STATUS.teams
```

Comment:
Is there anywhere else that `STATUS.teams` lives as a constant that we should be removing?